### ISSUE TITLE
CSS bugs in API documentation sidebar

### ISSUE THREAD

**Issue by olivierlacan** [rails/rails#39045]
### Steps to reproduce
Visit https://api.rubyonrails.org/ 

### Expected behavior
No styling issues.

### Actual behavior
Several styling issues: 
- on load bars appear overlaid on items in the sidebar but don't appear on every page load
- items listed in the sidebar are too tangent (missing margin/padding) to the left

![](http://link.olivierlacan.com/4a5977762fc7/Screen%252520Recording%2525202020-04-25%252520at%25252004.44.gif)

### System configuration
Google Chrome macOS Version 81.0.4044.122 (Official Build) (64-bit)

**Comment by simi** [comment]
https://github.com/rails/rails/issues/38768#issuecomment-601393775
https://github.com/rails/rails/issues/39005#issuecomment-617075924

TL;DR current suggestion is to use edgeapi.rubyonrails.org and wait patiently for next release for updated docs.

**Comment by eugeneius** [comment]
I'm using the same version of Chrome on macOS and I don't see those black bars 😕 do you get them on https://edgeapi.rubyonrails.org/ too?

As @simi mentioned, the overflow issue will be fixed in the next stable release.

**Comment by eugeneius** [comment]
6.0.3 was released today, and the margin problem is now resolved. 🎉

I'm going to close this, but let me know if the other problem is still present and we can investigate further.
